honeybee_3dm/face.py

```diff
@@ -0,0 +1,145 @@
+import rhino3dm
+import uuid
+from togeometry import to_face3d
```

Comment:
for relative import you should us . before the module name. Depending on the operating system this can fail.

```suggestion
from .togeometry import to_face3d
```

Reply:
Complied.